PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.h

```diff
@@ -107,32 +107,22 @@ class AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA : public AliAnalysisTaskEmcalJ
     //void                        DoMCJetLoop()                                     ;
     //Bool_t                      IsLEDEvent() const                                ;
     
-    Bool_t                      fUseRecalcPatches                                 ;///<                  Switch between offline (FEE) and recalc (L1) patches
+    Bool_t                      fUseRecalcPatches                                 ;//                  Switch between offline (FEE) and recalc (L1) patches
```

Comment:
This is as well a change without effect: This doxygen comment is transparent to both ROOT5 and ROOT6 and will therefor be ignored by rootcint/rootcling.